tracks/elm/exercises/twelve-days/mentoring.md

```diff
@@ -0,0 +1,41 @@
+---
+
+This is a good exercise to talk about naming, as it is quite difficult and most students struggle with it.
```

Comment:
Just because I'm interested 😄, is this a problem with this particular exercise, or is this the "naming things is hard" comment?

Reply:
Naming is hard, as you say! Some exercises are harder than others. In the `bob` exercise naming is relatively easy and most students do well. This is the next exercise in the list I think, and the naming is much harder (and most students do not do well). There are of course other exercises where the naming is hard ...